modules/gpgsqlbackend/spgsql.cc

```diff
@@ -77,7 +77,19 @@ class SPgSQLStatement: public SSqlStatement
   SSqlStatement* execute() {
     prepareStatement();
     if (d_dolog) {
-      g_log<<Logger::Warning<< "Query "<<((long)(void*)this)<<": " << d_query << endl;
+      g_log<<Logger::Warning<< "Query "<<((long)(void*)this)<<": Statement: " << d_query << endl;
```

Comment:
hmm, how about no `endl` here? Not a strong opinion but wanted to ask.

Reply:
I wanted this to look somewhat similar to the postgres server logs, which also puts statement and params on separate lines. Might also be nicer (or worse?) for grepping.

Reply:
Yes, goes both ways for grepping. I see that we use the same prefix (numberised `this`) for more than just these two lines, so splitting here is fine too.